content/en/blog/2023-06-01-may-update/index.md

```diff
@@ -0,0 +1,274 @@
+  helm-controller pods are marked as
+  [system-cluster-critical](https://kubernetes.io/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/).
+- The `Alert` v1beta2 API has two new optional fields. `.spec.inclusionList` for
+  fine-grained control over events filtering (notification-controller) and 
+  `.spec.metadata` that allows users to enrich the alerts with information 
+  about the cluster name, region, environment, etc.
+- New command `flux reconcile source chart` for pulling Helm OCI charts on-demand
+  from container registries (CLI).
+- Support annotated Git tags with .spec.ref.name in GitRepository (source-controller).
+- The deprecated field `.status.url` was removed from the `Receiver` v1
+  API (notification-controller).
+- Add support for commit signing using OpenPGP keys with
+  passphrases (image-automation-controller).
+- Fix bootstrap for BitBucket Server (CLI).
+- Fix secrets decryption when using Azure Key Vault (kustomize-controller).
+- Fix drift detection for renamed HelmReleases (helm-controller).
+- Improve performance when handling webhook receivers (notification-controller).
+- Improve the detection of values changes for HelmReleases by stable 
+  sorting them by key (helm-controller)
+- Update cosign to v2 (source-controller)
+- Support for Helm 3.12.0 and Kustomize v5.0.3.
+
+To upgrade from v0.x to v2.0.0-rc.5, please see
+[the procedure documented in RC.1.](https://github.com/fluxcd/flux2/releases/tag/v2.0.0-rc.1).
+
+:warning: Note that Kubernetes 1.27.0 contains a regression bug that affects
+Flux, it is recommended to upgrade Kubernetes to 1.27.1 or newer. The upgarde to
+Kustomize v5 also contains breaking changes, please consult their
+[changelog](https://github.com/kubernetes-sigs/kustomize/releases/tag/kustomize%2Fv5.0.0) for more details.
+
+Big thanks to all the Flux contributors that helped us with this release!
+
+### Security news
+
+All components have been updated to patch vulnerabilities in Docker (CVE-2023-28840,
+CVE-2023-28841, CVE-2023-28842) and Sigstore (CVE-2023-30551). The source-controller
+has also been updated to patch a vulnerability in Sigstore (CVE-2023-33199)
+
+### Flagger 1.31.0
+
+This release adds support for Linkerd 2.13. Furthermore, a bug which led the confirm-rollout
+webhook to be executed at every step of the Canary instead of only being executed before the
+canary deployment is scaled up, has been fixed.
+
+
+:warning: This release contains some breaking changes for the Linkerd integration.
+Please see the [Changelog](https://github.com/fluxcd/flagger/blob/main/CHANGELOG.md#1310)
+on how to upgrade.
+
+## News from Flux users & the Community!
+
+### Newly posted Flux adopters!
+
+{{< gallery match="logos/*" sortOrder="desc" rowHeight="150" margins="5"
+            thumbnailResizeOptions="600x600 q90 Lanczos"
+            previewType="blur" embedPreview=true lastRow="nojustify" >}}
```

Comment:
The rendered order of logos in here doesn't match the list below but I don't know how to fix that. If we can't find a way to change it, let's change the order of the list below to match the logos here.